KlarnaOnDemand/KODRegistrationViewController.m

```diff
@@ -43,9 +43,10 @@ - (NSURL *)url {
   return [KODUrl registrationUrlWithSettings: _registrationSettings];
 }
 
-- (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error {
-  [super webView:webView didFailLoadWithError:error];
-  
+
+-(void)webView:(WKWebView *)webView didFailNavigation:(WKNavigation *)navigation withError:(NSError *)error {
```

Comment:
for the sake of consistency, add a space between `-` and `(void)`. same goes to line 40 in `KODPreferencesViewController.m`, and lines 22, 47, 53 in `KODWebViewController.m`.
also, there's an extra newline up here